### PR TITLE
feat: adds the defaults readme and updates other readmes

### PR DESCRIPTION
**Pull request by carlossteinzup**
This PR adds the beagle default readme and updates minor adjustments on the general, android and beagle-scaffold readmes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
 # Beagle-helpers
 
-This is a repository to organize several auxiliary frameworks/libs made by our team.
+Beagle Helpers is a solution to facilitate the process of implementing a Beagle framework 
+into an Android, iOS or Web project
+
+The initial setup of Beagle requires quite a few classes and configurations and in order to make this process 
+faster and more dynamic each platform have developed a few libs.
```
Comment (hthemir) [inline README.md:7]:
```suggestion
The initial setup of Beagle requires quite a few classes and configurations and, in order to make this process 
faster and more dynamic, each platform has developed a few libs.
```

File: android/README.md [hunk 2]
```diff
@@ -11,16 +11,21 @@ steps, and starting studies quicker.
...
-### Getting started
 <hr>
 
+### Getting started
+
 To start using Beagle right now, you can use the libs below:
 * [Beagle-Scaffold](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold):
 This lib will hold almost all configurations needed to start using Beagle in your project.
-We advise using these libs for people that never used Beagle before.
+We advise to use these libs for people that never used Beagle before.
```
Comment (hthemir) [inline android/README.md:23]:
```suggestion
We advise the use of these libs to people that never used Beagle before.
```

File: android/beagle-defaults/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    override val baseUrl: String = "https://adopt-beagle.continuousplatform.com/scaffold"
+    override val isLoggingEnabled: Boolean = true
+    override val cache: Cache = Cache(
+        enabled = false,
+        maxAge = 300,
+        size = 15
+    )
+}
+```
+b) Create an HttpApp file as listed below. This class extends from an ´HttpClientDefault()´ class at the Beagle-Default lib:
```
Comment (hthemir) [inline android/beagle-defaults/README.md:75]:
```suggestion
b) Create a HttpApp file as listed below. This class extends from a ´HttpClientDefault()´ class at the Beagle-Default lib:
```

File: android/beagle-defaults/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+```
+@BeagleComponent
+class HttpApp: HttpClientDefault()
+```
+
+c) Create a CacheApp file as listed below. This class extends from an ´StoreHandlerDefault()´ class at the Beagle-Default lib:
```
Comment (hthemir) [inline android/beagle-defaults/README.md:81]:
```suggestion
c) Create a CacheApp file as listed below. This class extends from a ´StoreHandlerDefault()´ class at the Beagle-Default lib:
```

File: android/beagle-defaults/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+@BeagleComponent
+class CacheApp : StoreHandlerDefault(AppApplication.APPLICATION!!)
+```
+
+d) Create a LoggerApp file as listed below. This class extends from an ´BeagleLoggerDefault()´ class at the Beagle-Default lib:
```
Comment (hthemir) [inline android/beagle-defaults/README.md:87]:
```suggestion
d) Create a LoggerApp file as listed below. This class extends from a ´BeagleLoggerDefault()´ class at the Beagle-Default lib:
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,17 @@
 # Beagle-helpers
 
-This is a repository to organize several auxiliary frameworks/libs made by our team.
+Beagle Helpers is a solution to facilitate the process of implementing a Beagle framework 
+into an Android, iOS or Web project
```
Comment (luismaximozup) [inline README.md:4]:
```suggestion
into an Android, iOS or Web projects
```

**Review by luismaximozup — DISMISSED** [review]

File: README.md [hunk 7]
```diff
@@ -1,3 +1,17 @@
...
+
+The initial setup of Beagle requires quite a few classes and configurations and, in order to make this process 
+faster and more dynamic, each platform has developed a few libs.
```
Comment (vitorbassozup) [inline README.md:7]:
```suggestion
The initial setup of Beagle requires quite a few classes and configurations and, in order to make this process faster and more dynamic, each platform has developed a few libs.
```

File: README.md [hunk 8]
```diff
@@ -1,3 +1,17 @@
 # Beagle-helpers
 
-This is a repository to organize several auxiliary frameworks/libs made by our team.
+Beagle Helpers is a solution to facilitate the process of implementing a Beagle framework 
+into an Android, iOS or Web projects
```
Comment (vitorbassozup) [inline README.md:4]:
```suggestion
Beagle Helpers is a solution to facilitate the use of the Beagle framework in an Android, iOS or Web projects
```

File: README.md [hunk 9]
```diff
@@ -1,3 +1,17 @@
...
+
+The initial setup of Beagle requires quite a few classes and configurations and, in order to make this process 
+faster and more dynamic, each platform has developed a few libs.
+
+<hr>
+
+### How to configure?
+Here you will find each Android, iOS and WEB repository for libs or scripts that assists in implementing 
+a Beagle Framework in your project:
```
Comment (vitorbassozup) [inline README.md:13]:
```suggestion
Here you will find Android, iOS and WEB repositories for libs or scripts that assists in implementing a Beagle Framework in your project:
```

File: README.md [hunk 10]
```diff
@@ -1,3 +1,17 @@
...
+
+* [Android libs](https://github.com/ZupIT/beagle-helpers/tree/main/android)
+* [iOS libs](https://github.com/ZupIT/beagle-helpers/tree/main/iOS)
+* Web libs
```
Comment (vitorbassozup) [inline README.md:17]:
```suggestion
* [**Android libs**](https://github.com/ZupIT/beagle-helpers/tree/main/android)
* [**iOS libs**](https://github.com/ZupIT/beagle-helpers/tree/main/iOS)
* Web libs
```

File: android/README.md [hunk 11]
```diff
@@ -11,16 +11,20 @@ steps, and starting studies quicker.
 
 * If you wish to use Beagle for an application in production we advise you to configure a project
  from scratch using our
- [documentation](http://docs-beta.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)
+ [documentation](https://docs.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)
```
Comment (vitorbassozup) [inline android/README.md:14]:
```suggestion
 [**documentation**](https://docs.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)
```

File: android/README.md [hunk 12]
```diff
@@ -11,16 +11,20 @@ steps, and starting studies quicker.
...
 
-### Getting started
 <hr>
 
+### Getting started
+
 To start using Beagle right now, you can use the libs below:
 * [Beagle-Scaffold](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold):
 This lib will hold almost all configurations needed to start using Beagle in your project.
```
Comment (vitorbassozup) [inline android/README.md:22]:
```suggestion
To start using Beagle right now, you can use the libs below:
* [**Beagle-Scaffold**](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold)
This lib will take care of almost all configurations needed to start using Beagle in your project.
```

File: android/README.md [hunk 13]
```diff
@@ -11,16 +11,20 @@ steps, and starting studies quicker.
...
-We advise using these libs for people that never used Beagle before.
+We advise the use of these libs for people that never used Beagle before.
```
Comment (vitorbassozup) [inline android/README.md:23]:
```suggestion
We recommend the use of this lib for people that are just starting to learn Beagle.
```

File: android/README.md [hunk 14]
```diff
@@ -11,16 +11,20 @@ steps, and starting studies quicker.
...
 
 * [Beagle-Defaults](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-defaults):
```
Comment (vitorbassozup) [inline android/README.md:25]:
```suggestion
* [**Beagle-Defaults**](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-defaults):
```

File: android/README.md [hunk 15]
```diff
@@ -11,16 +11,20 @@ steps, and starting studies quicker.
...
-This lib is meant for more experienced Beagle users and will hold only a few class configurations necessary to use Beagle in an application. These configurations include default HttpClient, Logger and Cache classes. All these are also available in the Scaffold lib above.
-
+This lib is advised for more experienced Beagle users and 
+will hold only a few class configurations necessary to 
+use Beagle in an application. These configurations include 
+default HttpClient, Logger and Cache classes.<br>
+All these are also available in the Scaffold lib above.
```
Comment (vitorbassozup) [inline android/README.md:30]:
```suggestion
This lib is advised for more experienced Beagle users as it only takes care of a few class configurations necessary to use Beagle in an application. These configurations include a default HttpClient, Logger and Cache classes.<br> 
All these are also available in the Scaffold lib above.
```

File: android/beagle-defaults/README.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
+# Beagle Scaffold Android
+<hr>
+
+![Maven Central](https://img.shields.io/maven-central/v/br.com.zup.beagle/beagle-defaults?color=green&label=Beagle-Defaults)
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:4]:
```suggestion
![**Maven Central**](https://img.shields.io/maven-central/v/br.com.zup.beagle/beagle-defaults?color=green&label=Beagle-Defaults)
```

File: android/beagle-defaults/README.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+![Maven Central](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:5]:
```suggestion
![**Maven Central**](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)
```

File: android/beagle-defaults/README.md [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+
+Here you will find the **Beagle-Defaults** library to help you start a project using Beagle in Android.
+This lib will hold just a few classes necessary 
+to configure and add Beagle into an Android project.
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:9]:
```suggestion
This lib will hold just a few classes necessary to configure and add Beagle into an Android project.
```

File: android/beagle-defaults/README.md [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+
+### Requirements
+Before you start to configure Beagle for your Android system, it’s important to check out if you have installed all the current versions of the following programs: ‌
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:12]:
```suggestion
Before you start configuring Beagle for your Android system, it’s important to check if you have installed all the current versions of the following dependencies: ‌
```

File: android/beagle-defaults/README.md [hunk 20]
```diff
@@ -0,0 +1,128 @@
...
+* JDK 8+ language
+* Android API level: at least 19 or higher
+* Kotlin 1.3+
+
+### Getting started
+<hr>
+
+This tutorial will configure Beagle from the Start.
+
+Step 1: Add a configuration on the Gradle file in your project:
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:23]:
```suggestion
Step 1: Add a configuration to the Gradle file in your project:
```

File: android/beagle-defaults/README.md [hunk 21]
```diff
@@ -0,0 +1,128 @@
...
+    repositories {
+        google()
+        jcenter()
+        mavenCentral()
+    }
+}
+```
+Step 2: Add kotlin kapt plugin and 3 dependencies on your application dependency manager (gradle.gradle)<br>
+
+ ![Maven Central](https://img.shields.io/maven-central/v/br.com.zup.beagle/beagle-scaffold?label=beagle-scaffod)
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:37]:
```suggestion
 ![**Maven Central**](https://img.shields.io/maven-central/v/br.com.zup.beagle/beagle-scaffold?label=beagle-scaffod)
```

File: android/beagle-defaults/README.md [hunk 22]
```diff
@@ -0,0 +1,128 @@
...
+ ![Maven Central](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)
```
Comment (vitorbassozup) [inline android/beagle-defaults/README.md:38]:
```suggestion
 ![**Maven Central**](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)                                                                   
```

File: android/beagle-defaults/README.md (hunk 3 above)
Comment (vitorbassozup) [reply android/beagle-defaults/README.md:75]: I believe the use of an instead of a here is correct.

File: android/beagle-scaffold/README.md [hunk 23]
```diff
@@ -5,7 +5,7 @@
 ![Maven Central](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)
```
Comment (vitorbassozup) [inline android/beagle-scaffold/README.md:5]:
```suggestion
![**Maven Central**](https://img.shields.io/maven-central/v/br.com.zup.beagle/android?label=Beagle)
```

**Comment by botbeaglezup** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: dlopen(/Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.7.dylib
  Referenced from: /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
  Reason: image not found - /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
```
 #  from danger/android/Dangerfile:14
 #  -------------------------------------------
 #    android_lint.report_file = file_name
 >    android_lint.lint(inline_mode: true)
 #  end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: dlopen(/Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.7.dylib
  Referenced from: /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
  Reason: image not found - /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
```
 #  from danger/android/Dangerfile:14
 #  -------------------------------------------
 #    android_lint.report_file = file_name
 >    android_lint.lint(inline_mode: true)
 #  end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: dlopen(/Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.7.dylib
  Referenced from: /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
  Reason: image not found - /Users/runner/.gem/ruby/2.7.0/gems/ruby-ll-2.1.2/lib/libll.bundle
```
 #  from danger/android/Dangerfile:14
 #  -------------------------------------------
 #    android_lint.report_file = file_name
 >    android_lint.lint(inline_mode: true)
 #  end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by gabrielleonardozup — APPROVED** [review]

File: android/beagle-defaults/README.md [hunk 24]
```diff
@@ -0,0 +1,128 @@
+# Beagle Scaffold Android
```
Comment (jacoabreuzup) [inline android/beagle-defaults/README.md:1]: Change the title to Beagle Defaults Defaults

**Review by gabrielleonardozup — APPROVED** [review]

**Pull request merged**